apps/worker/src/app/workflow/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -322,7 +337,8 @@ export class SendMessageChat extends SendMessageBase {
         content,
       });
 
-      await this.createExecutionDetails.execute(
+      await this.executionLogQueueService.add(
+        command.jobId,
```

Comment:
This would need to be a unique value for later easier debugging, could you just pass here a random value?

Reply:
okay. may be i can generate and pass the mongo _id here, which could help in relating the data with event, but do you think would be benefitial? i'm not sure what kind of debugging you are talking here, because if its just for failures then anyway we wont have the data stored it will be useless.